File: Examples/Sample-ObjC/Sample-ObjC/ViewController.m
//
//  ViewController.m
//  Sample-ObjC
//
//  Created by Marc Bauer on 19.02.20.
//  Copyright © 2020 Crisp IM SAS. All rights reserved.
//

#import "ViewController.h"
#import <Crisp/Crisp.h>

@interface ViewController ()
@end

@implementation ViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  // Do any additional setup after loading the view.
}

- (IBAction)presentChat:(id)sender {
  [self presentViewController:[[CRSPChatViewController alloc] init] animated:YES completion:nil];
}
@end

File: Examples/Sample-Swift/Sample-Swift/ViewController.swift
//
//  ViewController.swift
//  Sample-Swift
//
//  Created by Marc Bauer on 19.02.20.
//  Copyright © 2020 Crisp IM SARL. All rights reserved.
//

import UIKit
import Crisp

class ViewController: UIViewController {

  override func viewDidLoad() {
    super.viewDidLoad()
    // Do any additional setup after loading the view.
  }

  @IBAction func presentChat(_ sender: Any) {
    self.present(ChatViewController(), animated: true)
  }
}
